bot.py

```diff
@@ -0,0 +1,233 @@
+import github
+
+
+def add_coll(h):
+    f = 0
+    while f == 0:
+        print("Must I add any collaborator to this repo? (Yes/No)")
+        d = input()
+        if d == "No":
```

Comment:
For the future, check all answers in lower case. Because "yes" or "YEs" command your bot can't understant.